[functorch] eliminate erroneous warning on `import functorch`

zou3519 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #82905

Warning comes from registering an operator twice to a dispatch key

Test Plan:
- `import functorch`

zou3519 committed:
[functorch] eliminate erroneous warning on `import functorch`

Warning comes from registering an operator twice to a dispatch key

Test Plan:
- `import functorch`

[ghstack-poisoned]

Chillee reviewed (approved):
ah I've been seeing this :P

zou3519 committed:
Update on "[functorch] eliminate erroneous warning on `import functorch`"


Warning comes from registering an operator twice to a dispatch key

Test Plan:
- `import functorch`

[ghstack-poisoned]

zou3519 commented:
@pytorchbot merge -f "CI ran already, needed to unstack this PR in order to land it due to failures lower down in the stack"
